users/models.py

```diff
@@ -68,3 +69,34 @@ class User(AbstractBaseUser, PermissionsMixin):
 
     def __str__(self):
         return self.email
+
+
+class Owner(models.Model):
+    user = models.OneToOneField(AUTH_USER_MODEL, on_delete=models.CASCADE, primary_key=True)
+
+    def __str__(self):
+        return self.user.email
+
+
+class City(models.Model):
+    cityName = models.CharField(max_length=100)
+
+    def __str__(self):
+        return self.cityName
+
+
+class Apartment(models.Model):
+    owner = models.ForeignKey(Owner, on_delete=models.CASCADE, related_name='owner_apartments')
+    datePosted = models.DateTimeField(default=timezone.now, blank=True)
+    city = models.ForeignKey(City, on_delete=models.RESTRICT, related_name='city_apartments')
+    address = models.TextField()
+    rentPricePerMonth = models.IntegerField()
+    numOfRoomates = models.IntegerField()
+    numOfRooms = models.IntegerField()
+    startDate = models.DateField()
+    content = models.TextField(blank=True)
+    isRelevant = models.BooleanField(default=True, blank=True)
```

Comment:
What is the purpose of the isRelevant field? Is this for the owner to mark that it shouldn't be shown, or for some other purpose?

Reply:
Yes, I thought that if the owner found a partner he could mark that the apartment is no longer relevant and then hide it from the search but keep it in history.
In addition, if after a certain time he is looking for another partner with the same apartment, he can return it from history